### PR TITLE
Better gazettes integration

### PR DESCRIPTION
**Pull request by nickmwangemi**
https://www.loom.com/share/3169933e25de43ddbad64a579b57be9d

### CODE REVIEW

File: lawlibrary/static/stylesheets/lawlibrary.scss [hunk 1]
```diff
@@ -9,3 +9,26 @@ footer {
 .btn.btn-primary {
   color: #fff;
 }
+
```
Comment (longhotsummer) [inline lawlibrary/static/stylesheets/lawlibrary.scss:12]: All the views and templates will be used by other LIIWeb websites, so need to be in the liiweb app, not lawlibrary.

File: peachjam/models/gazette.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from peachjam.models import CoreDocument
+
+
+class Gazette(CoreDocument):
+    pass
```
Comment (longhotsummer) [inline peachjam/models/gazette.py:5]: need to set `doc_type` in `save()`

File: lawlibrary/urls.py [hunk 3]
```diff
@@ -36,5 +37,17 @@
         views.ProvincialLegislationListView.as_view(variant="all"),
         name="provincial_legislation_list_all",
     ),
+    path("gazettes", views.GazetteListView.as_view(), name="gazettes"),
```
Comment (longhotsummer) [inline lawlibrary/urls.py:40]: this can move into liiweb

File: lawlibrary/urls.py [hunk 4]
```diff
@@ -36,5 +37,17 @@
...
+    path(
+        "gazettes/<str:code>/",
+        views.ProvincialGazetteListView.as_view(),
+        name="provincial_gazette_list",
+    ),
+    path("gazettes/<int:year>", views.YearView.as_view(), name="gazettes_by_year"),
```
Comment (longhotsummer) [inline lawlibrary/urls.py:46]: this can move into liiweb

**Comment by longhotsummer** [comment]
URLS:

LIIWeb (generic, no place codes):
* ulii.org/gazettes/
* ulii.org/gazettes/2022/

Lawlibrary-specific:
* lawlibrary.org.za/gazettes/za-wc/
* lawlibrary.org.za/gazettes/za-wc/2022

**Comment by longhotsummer** [comment]
I've pushed some changes to move the gazette views into peach jam, and just override them slightly in lawlibrary. I suspect in future we'll want the locality-based views in peachjam too, but let's go with this for now.

We could safely merge this now.

The final aspect, which can be added separately, is an ingestor adapter to fetch gazettes from another peachjam website (gazettes.africa) by reading from the database.

**Review by longhotsummer — DISMISSED** [review]

**Review by actlikewill — APPROVED** [review]
Works well.

**Review by longhotsummer — APPROVED** [review]

**Review by actlikewill — APPROVED** [review]

**Pull request merged**